### PR TITLE
Extend test postgres resources with Rbac and PSP

### PR DESCRIPTION
**Pull request by ubergesundheit**
This PR changes the postgres test resources to include RBAC, ServiceAccount and PSP for easier usage on GS workload clusters

### CODE REVIEW

**Comment by webwurst** [comment]
Does this need to be a `ClusterRole`? Polutes the glogal space and makes updates harder :)

File: tests/ats/postgres.yaml [hunk 1]
```diff
@@ -19,6 +19,87 @@ spec:
   selector:
     app: postgres
 ---
+kind: ClusterRole
```
Comment (ubergesundheit) [inline tests/ats/postgres.yaml:22]:
```suggestion
kind: Role
```

File: tests/ats/postgres.yaml [hunk 2]
```diff
@@ -19,6 +19,87 @@ spec:
...
+- apiGroups:
+  - policy
+  resources:
+  - podsecuritypolicies
+  resourceNames:
+  - postgres
+  verbs:
+  - use
+---
+kind: ClusterRoleBinding
```
Comment (ubergesundheit) [inline tests/ats/postgres.yaml:39]:
```suggestion
kind: RoleBinding
```

File: tests/ats/postgres.yaml [hunk 3]
```diff
@@ -19,6 +19,87 @@ spec:
...
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  labels:
+    app: postgres
+  name: postgres
+  namespace: postgres
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: postgres
```
Comment (ubergesundheit) [inline tests/ats/postgres.yaml:49]:
```suggestion
  kind: Role
  name: postgres
  namespace: postgres
```

**Comment by ubergesundheit** [comment]
lets try in 39a1b5c34f57098b8524396f5d65080ce1844008

File: tests/ats/postgres.yaml [hunk 4]
```diff
@@ -19,6 +19,88 @@ spec:
...
+metadata:
+  labels:
+    app: postgres
+  name: postgres
+  namespace: postgres
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: postgres
+  namespace: postgres
```
Comment (ubergesundheit) [inline tests/ats/postgres.yaml:50]:
```suggestion
```

**Pull request merged**